identity/src/main/java/com/android/identity/DataTransportBle.java

```diff
@@ -258,6 +260,10 @@ Pair<NdefRecord, byte[]> buildNdefRecords(boolean centralClientSupported,
         return new Pair<>(record, acRecordPayload);
     }
 
+    public void setSupportL2CAP(boolean mSupportL2CAP) {
```

Comment:
I would call this `setUseL2CAPIfAvailable()` instead.

Reply:
Done